template/apps/thanos.json

```diff
@@ -2,20 +2,20 @@
 	"categories": [
 		"Other",
 		"Tools",
-		"Minecraft"
+		"Games"
 	],
   "env": [
   	{
   		"name": "TZ",
   		"label": "TZ",
-  		"default": "America/New_York",
-  		"description": "Specify a timezone to use for example Europe/London"
+  		"default": "Europe/London",
```

Comment:
I mean, London is the best city, but there wasn't anything wrong with the format of `America/New_York` ?

Reply:
```json 
 		"default": "Europe/London",
  		"description": "Specify a timezone to use. For example, Europe/London"
```
I just wanted it to match the description, if there's an issue i'll change it back